pkg/aws/client_test.go

```diff
@@ -1,13 +1,16 @@
 package aws_test
 
 import (
+	"fmt"
+	aws2 "github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	tags2 "github.com/openshift/rosa/pkg/aws/tags"
```

Comment:
NP: personally I'd rather name the import tags and the variable something else

Reply:
+1 this, same for `aws2`

Reply:
@gdbranco @ciaranRoche I'm not assigning a variable using these packages, only relying on a function from them. There is a clash between the ROSA `aws` package and the AWS SDK `aws` package. I'll rename to the following:

- `aws2` --> `awsSdk`
- `tags2` --> `rosaTags`

Let me know if that works for you 👍